Extensions/Guardrail+Settings.swift

```diff
@@ -138,12 +138,11 @@ public extension Guardrail where Value == HKQuantity {
     static func maximumBolus(supportedBolusVolumes: [Double]) -> Guardrail {
         let maxBolusThresholdUnits: Double = 30
         let maxBolusWarningThresholdUnits: Double = 20
-        let minimumSupportedBolusVolume = supportedBolusVolumes.first!
-        let recommendedLowerBound = minimumSupportedBolusVolume == 0 ? supportedBolusVolumes.dropFirst().first! : minimumSupportedBolusVolume
+        let supportedBolusVolumes = supportedBolusVolumes.drop { $0 == 0 }
```

Comment:
[nit] Would using `<= 0` be too paranoid?

Reply:
Good point... I thought of that.  I don't check for negatives anywhere else, so I left it as-is. 🤷‍♂️

Reply:
Actually, on second thought, I'll put a check in there just in case (and a unit test)